src/index.ts

```diff
@@ -212,6 +212,16 @@ function resolveLaravelPlugin(pluginConfig: Required<PluginConfig>): LaravelPlug
                 if (isAddressInfo(address)) {
                     viteDevServerUrl = userConfig.server?.origin ? userConfig.server.origin as DevServerUrl : resolveDevServerUrl(address, server.config, userConfig)
 
+                    const hotFileParentDirectory = path.dirname(pluginConfig.hotFile);
+
+                    if (! fs.existsSync(hotFileParentDirectory)) {
+                        fs.mkdirSync(hotFileParentDirectory, { recursive: true })
+
+                        setTimeout(() => {
+                            server.config.logger.info(`${colors.green('laravel')} Hot file directory created ${colors.dim(fs.realpathSync(hotFileParentDirectory))}`, { clear: true, timestamp: true })
+                        }, 200)
```

Comment:
This timeout is intentional to ensure our log appears after the initial Vite output. We use a similar technique below.